code/modules/mob/living/say.dm

```diff
@@ -96,18 +98,33 @@ var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
 		src << "<span class='warning'>You find yourself unable to speak!</span>"
 		return
 
-	if(message_mode != MODE_WHISPER) //whisper() calls treat_message(); double process results in "hisspering"
-		message = treat_message(message)
+	var/message_range = 6

```

Comment:
Yes, but since you can eavesdrop on everything 1 tile above the specified range, so I made it 6 so that when you "eavesdrop" you're still on screen.